reference/configuration/framework.rst

```diff
@@ -899,18 +949,23 @@ Now you can use the ``avatars`` package in your templates:
 
 Each package can configure the following options:
 
-* :ref:`base_urls <reference-templating-base-urls>`
-* :ref:`version <reference-framework-assets-version>`
-* :ref:`version_format <reference-templating-version-format>`
+* :ref:`base_urls <reference-assets-base-urls>`
+* :ref:`version <reference-assets-version>`
+* :ref:`version_format <reference-assets-version-format>`
 
 .. _reference-framework-assets-version:
 .. _ref-framework-assets-version:
```

Comment:
should be `reference-assets-version` I think - line 953 above is a broken link right now